pkg/maps/lbmap/affinity.go

```diff
@@ -28,14 +28,18 @@ var (
 
 // initAffinity creates the BPF maps for implementing session affinity.
 func initAffinity(params InitParams) {
+	if params.AffinityMapMaxEntries != DefaultMaxEntries {
```

Comment:
Is this check needed? We should just assume that whatever value is assigned to `params.AffinityMapMaxEntries` is the value that the user wants to use (ditto for all other cases)